study.md

```diff
@@ -135,15 +135,17 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a breif description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP or  hyptertext transfer protocol is what the browswer uses to recieve
+a server address and translate the message from that sever into text in the browser
+window.
 ```
```

Comment:
It's not necessarily translating anything, it's just the set of rules that these requests and responses follow, like a shared language, that clients and servers communicate with.